<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquationWidgetUI</class>
 <widget class="QWidget" name="EquationWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Equation</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="functionLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameters</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>31</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>–ê:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>B:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Epsilon:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>130</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="endLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>130</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="epsilonLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>130</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Result</string>
     </property>
     <widget class="QLineEdit" name="resultLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="solve">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plot">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>94</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <zorder>resultLineEdit</zorder>
     <zorder>solve</zorder>
     <zorder>clear</zorder>
     <zorder>plot</zorder>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Method</string>
     </property>
     <widget class="QComboBox" name="methods">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="statusTip">
    <string>Inserts a new Row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current Row</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
